lib/google/gax/api_callable.rb

```diff
@@ -239,21 +239,21 @@ def create_api_call(func, settings, params_extractor: nil)
         api_caller = bundleable(settings.bundle_descriptor)
       end
 
-      proc do |request, options = nil|
+      proc do |request, options = nil, &block|
         this_settings = settings.merge(options)
         if params_extractor
           params = params_extractor.call(request)
           this_settings = with_routing_header(this_settings, params)
         end
         api_call = if settings.retry_codes?
                      retryable(func, this_settings.retry_options,
-                               this_settings.kwargs)
+                               this_settings.metadata)
```

Comment:
Why is this change happening? This seems like a breaking change.

Reply:
I think the `merge!` should avoid breaking current users, right?

Context on change: https://github.com/googleapis/gax-ruby/issues/86

Reply:
Yes, I kept the merge in to avoid breaking it.